administrator/components/com_templates/Model/TemplateModel.php

```diff
@@ -197,8 +197,24 @@ public function getUpdatedList($state = false, $all = false, $cleanup = false)
 			$client = ApplicationHelper::getClientInfo($pk->client_id);
 			$path = Path::clean($client->path . '/templates/' . $pk->template . base64_decode($pk->hash_id));
 
+			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
+
 			if (file_exists($path))
 			{
+				if ((int) $pk->created_date)
+				{
+					$created_date = new Date($pk->created_date);
+					$created_date->setTimezone($tz);
+					$pk->created_date = $created_date->toSql(true);
```

Comment:
`toSql(true)` means you're getting the timestamp in the correct SQL format but localized, so this is converting from UTC to whatever timezone you're setting above.

Look at the `check` methods of various `Joomla\CMS\Table\Table` subclasses for how created and modified dates are handled.

Reply:
@mbabker Thank you for your help. But if I do not set the timezone, the date will be displayed in UTC.

Reply:
Sorry, missed this wasn't a save operation.

The timezone shouldn't be normalized in models.  This should be done in views/layouts.  So you should use the `JHtml::date()` helper (as is done elsewhere in core) for this, as that will adjust the timezone for the user's configuration.